docs/man_pages/lib-management/plugin-add.md

```diff
@@ -43,15 +45,16 @@ Enables <%=plugins%> for your project. <% if(isHtml) { %>If the plugin has plugi
 <% if((isConsole && (isCordova || isNativeScript)) || isHtml) { %>
 ### Options
 * `--available` - Lists all <%=plugins%> that you can enable in your project and shows information about their variables, if any.
```

Comment:
* `--available` - Lists all <%=plugins%> that you can enable in your project and shows information about their variables, if any.<% if(isNativeScript) { %> For NativeScript apps, you can set the  `--count` option to specify the number of npm and NativeScript modules that will be listed.<% } %>